hardware/Makefile

```diff
@@ -76,8 +88,8 @@ vlog_args += -suppress vlog-2583 -suppress vlog-13314 -suppress vlog-13233
 vlog_args += -work $(library)
 # Defines
 vlog_defs += -DNUM_CORES=$(num_cores) -DNUM_CORES_PER_TILE=$(num_cores_per_tile)
-vlog_defs += -DL2_BASE="32'h$(l2_base)" -DL2_SIZE="32'h$(l2_size)"
-vlog_defs += -DBOOT_ADDR="32'h$(boot_addr)" -DXPULPIMG="1'b$(xpulpimg)"
+vlog_defs += -DL2_BASE="32\'h$(l2_base)" -DL2_SIZE="32\'h$(l2_size)"
+vlog_defs += -DBOOT_ADDR="32\'h$(boot_addr)" -DXPULPIMG=$(xpulpimg)
```

Comment:
Do we need this just temporarily until Bender is updated or even with the fixes in Bender?

Reply:
We could probably get rid of this when Bender gets updated. I made this fix the smallest possible one that makes Bender work with all our three targets.

Reply:
Perfect :+1: